src/reporters/tiny.js

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+
+export default report;
+
+async function report(runningTests, startTime) {
+  console.log("");
+  const summary = await reportSequentially(runningTests);
+  const { failed, passed, endTime } = summary;
+  const ok = color.green(`✓ ${passed}`);
+  const notOk = failed ? color.red(`✗ ${failed}`) : "";
+  const runTime = color.grey(`Ran in ${endTime - startTime}ms`);
+  console.log(`${ok} ${notOk} ${runTime}`);
+}
+
+async function reportSequentially(runningTests) {
+  let failed = 0;
+  for await (const result of runningTests) {
+    if (result.error) {
+      failed++;
+      console.log(formatResult(result));
+    }
+  }
+  return { failed, passed: runningTests.length - failed, endTime: Date.now() };
+}
+
+function formatResult(result) {
+  const { error, title } = result;
+  const { message, stack, operator } = error;
+  const header = color.red(`✗ ${title}`);
+  const path = color.grey(getFilePath(stack));
+  const errorMsg = formatMessage(message, operator);
+  return `${header}\n${path}\n${errorMsg}\n`;
+}
+
+function getFilePath(stack) {
+  const absolute = stack.match(/^ *at.*\(file:\/\/(.*?)\)/m)[1];
+  return path.relative(process.cwd(), absolute);
+}
+
+function formatMessage(message, operator) {
+  if (operator === "deepStrictEqual") {
```

Comment:
nicely done with the "operator" I would extract them to constants maybe?

`const DEEP_STRICT_EQUAL = 'deepStrictEqual';`

Reply:
I too look very suspiciously at hardcoded values and turn them into constants whenever possible, but in this case since I don't own the value (it comes from node assert), I'm only using it once, and the name is semantic enough... I think it kind of makes sense to keep it as is for now.